gulpfile.js/config.js

```diff
@@ -105,7 +106,7 @@ module.exports = {
   },
   
   images: {
-    govuk: govuk.images + '**/*',
+    govuk: [govuk.images, govuk.cssimgs].map(function (folder) { return folder  + '**/*'; }),
```

Comment:
no magic please, just hard code these, makes it tricky to have ones without, `'**/*'`